frame/contracts/src/weights.rs

```diff
@@ -171,3809 +172,3907 @@ pub trait WeightInfo {
+	/// Proof: Contracts CodeStorage (max_values: None, max_size: Some(126001), added: 128476, mode: Measured)
 	/// The range of component `c` is `[0, 61717]`.
 	fn reinstrument(c: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `270 + c * (1 ±0)`
-		//  Estimated: `0`
-		// Minimum execution time: 27_539 nanoseconds.
-		Weight::from_ref_time(23_554_889)
-			// Standard Error: 56
-			.saturating_add(Weight::from_ref_time(46_766).saturating_mul(c.into()))
+		//  Estimated: `3025 + c * (2 ±0)`
+		// Minimum execution time: 29_874 nanoseconds.
+		Weight::from_parts(34_405_679, 3025)
+			// Standard Error: 42
+			.saturating_add(Weight::from_ref_time(54_747).saturating_mul(c.into()))
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
+			.saturating_add(Weight::from_proof_size(2).saturating_mul(c.into()))
 	}
 	/// Storage: Contracts ContractInfoOf (r:1 w:1)
-	/// Proof: Contracts ContractInfoOf (max_values: None, max_size: Some(258), added: 2733, mode: Ignored)
+	/// Proof: Contracts ContractInfoOf (max_values: None, max_size: Some(258), added: 2733, mode: Measured)
 	/// Storage: Contracts CodeStorage (r:1 w:0)
-	/// Proof: Contracts CodeStorage (max_values: None, max_size: Some(126001), added: 128476, mode: Ignored)
+	/// Proof: Contracts CodeStorage (max_values: None, max_size: Some(126001), added: 128476, mode: Measured)
 	/// Storage: Timestamp Now (r:1 w:0)
-	/// Proof: Timestamp Now (max_values: Some(1), max_size: Some(8), added: 503, mode: Ignored)
+	/// Proof: Timestamp Now (max_values: Some(1), max_size: Some(8), added: 503, mode: Measured)
 	/// Storage: System Account (r:1 w:1)
-	/// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: Ignored)
+	/// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: Measured)
 	/// Storage: System EventTopics (r:2 w:2)
-	/// Proof Skipped: System EventTopics (max_values: None, max_size: None, mode: Ignored)
+	/// Proof Skipped: System EventTopics (max_values: None, max_size: None, mode: Measured)
 	/// The range of component `c` is `[0, 125952]`.
 	fn call_with_code_per_byte(c: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `771`
-		//  Estimated: `0`
-		// Minimum execution time: 297_710 nanoseconds.
-		Weight::from_ref_time(307_327_529)
-			// Standard Error: 18
-			.saturating_add(Weight::from_ref_time(29_849).saturating_mul(c.into()))
+		//  Estimated: `16780 + c * (5 ±0)`
+		// Minimum execution time: 346_882 nanoseconds.
+		Weight::from_parts(371_435_161, 16780)
+			// Standard Error: 25
+			.saturating_add(Weight::from_ref_time(34_745).saturating_mul(c.into()))
 			.saturating_add(T::DbWeight::get().reads(6_u64))
 			.saturating_add(T::DbWeight::get().writes(4_u64))
+			.saturating_add(Weight::from_proof_size(5).saturating_mul(c.into()))
```

Comment:
Oof. `5` bytes per byte of code. That is bad. It is probably due to our instrumentation worst case. We really need to get rid of that and move that into wasmi.